WalletWasabi/WabiSabi/Backend/Models/WabiSabiProtocolErrorCode.cs

```diff
@@ -21,8 +21,8 @@ public enum WabiSabiProtocolErrorCode
 		InputBanned,
 		InputNotWhitelisted,
 		AliceNotFound,
-		IncorrectRequestedWeightCredentials,
-		TooMuchWeight,
+		IncorrectRequestedVsizeCredentials,
+		TooMuchVsize,
```

Comment:
This error name sucks.

Reply:
there's many inconsistencies in these error code names and a few other things, this is one of the reasons I wanted to re-review everything we've done in the last few weeks, it seems like a more productive way to ensure consistency than to have back and forth on each PR separately

Reply:
copied!